pkg/helpers/azure/blob.go

```diff
@@ -0,0 +1,27 @@
+package azure
+
+import (
+	"os"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+	"github.com/pkg/errors"
+)
+
+func GetBlobSharedKey() (*azblob.SharedKeyCredential, error) {
+	storageAccountName, exist := os.LookupEnv("AZURE_STORAGE_ACCOUNT")
+	if !exist {
+		return nil, errors.New("AZURE_STORAGE_ACCOUNT should be defined to be able to read state from azure backend")
+	}
+
+	storageAccountKey, exist := os.LookupEnv("AZURE_STORAGE_KEY")
```

Comment:
I wanted to follow the [azure standard](https://docs.microsoft.com/en-us/cli/azure/storage/account?view=azure-cli-latest#az-storage-account-generate-sas-optional-parameters) 

Do you think I should add those flags (`--account-key` and `--account-name`) in addition of env vars ?

Reply:
Maybe we'll need to namespace them `--azure-account-key` and `--azure-account-name`, but they will no be standard anymore.

WDYT @sjourdan ?

Reply:
Yes that's what I have in mind actually, 2 flags that we could reuse as `DCTL_AZURE_...` just like we did for the terraform cloud one.

Reply:
Do you think we should prefix azure env vars with `DCTL_` ? 
I hate when I can't reuse SDK default configuration in third party tooling personally

Reply:
The `DCTL_` prefix will come from our flags (e.g. `bindEnvToFlags`). It's only for the container storage account and storage key I meant

Reply:
I think we should bind default azure env var to those flags, it's really annoying when you have to duplicate your env vars just because tooling is not using the same conventions.

I'm not against the flags, even if I don't find them really useful since we can use env vars. 
But at least they should be bound to default azure env vars `AZURE_STORAGE_ACCOUNT` and `AZURE_STORAGE_KEY` IMO, let me try to do that.

Reply:
I changed the approach, and removed `blob.go` helper function, WDYT of that [b1d8d0a](https://github.com/snyk/driftctl/pull/1357/commits/b1d8d0a5c614fc0ef715371858b8461a04618c2e) @wbeuil ?

Reply:
I'm good with that approach so that we don't have `DCTL_*` env variable working and we can have both your 2 envs and flags working as expected!

Reply:
> I'm good with that approach so that we don't have DCTL_* env variable working and we can have both your 2 envs and flags working as expected!

To be honest I didn't tested `DCTL_` flags but I think they may work too.

Reply:
I tested and it does not work with that:

```
if opts.StorageKey == "" || opts.StorageAccount == "" {
		return nil, errors.New("AZURE_STORAGE_ACCOUNT and AZURE_STORAGE_KEY should be defined to be able to read state from azure backend")
	}
```